packages/devtools-plugin/src/index.ts

```diff
@@ -12,6 +12,10 @@ export interface DevToolsUniverse<State> {
   state: State;
 }
 
+const serialize = (data: any) => {
```

Comment:
I'm not sure `serialize` is the right word, as serializing normally implies converting to a string, but this (may) turn it back into the original data. Instead could you serialize the entire message before calling `postMessage`? and then deserialize on the other side.